### PR TITLE
[#4064] Fix GitHub and resources

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Change GitHub and resources to ASF ones. Still to fix are Docker and the documentation URL. But this can be merged now.

### Why are the changes needed?

As we are now an ASF project.

Fix: #4064

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Built locally with no issues.

### CODE REVIEW

File: dev/docker/ranger/ranger-dependency.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ ranger_dir="$(cd "${ranger_dir}">/dev/null; pwd)"
 # Environment variables definition
 RANGER_VERSION=2.4.0
 RANGER_PACKAGE_NAME="ranger-${RANGER_VERSION}-admin.tar.gz" # Must export this variable for Dockerfile
-RANGER_DOWNLOAD_URL=https://github.com/datastrato/apache-ranger/releases/download/release-ranger-${RANGER_VERSION}/ranger-${RANGER_VERSION}-admin.tar.gz
+RANGER_DOWNLOAD_URL=https://github.com/apache/apache-ranger/releases/download/release-ranger-${RANGER_VERSION}/ranger-${RANGER_VERSION}-admin.tar.gz
```
Comment (jerryshao) [inline dev/docker/ranger/ranger-dependency.sh:27]: @xunliu can you please confirm that it is a correct change?

Comment (justinmclean) [reply dev/docker/ranger/ranger-dependency.sh:27]: That change is probably incorrect, but we may need to change how we obtain apache ranger. Why are we not getting it from teh ASF repo?

Comment (xunliu) [reply dev/docker/ranger/ranger-dependency.sh:27]: > That change is probably incorrect, but we may need to change how we obtain apache ranger. Why are we not getting it from teh ASF repo?

Apache Ranger didn't provide a release download, https://github.com/apache/ranger/releases.
So we compiled it myself and uploaded it to the datastrato/apache-ranger repository.

Comment (justinmclean) [reply dev/docker/ranger/ranger-dependency.sh:27]: GitHub is not an official release area for ASF projects. The official releases are put on https://dist.apache.org/. For Ranger see https://dist.apache.org/repos/dist/release/ranger/.

File: build.gradle.kts [hunk 2]
```diff
@@ -388,7 +388,7 @@ subprojects {
             }
           }
           scm {
-            url.set("https://github.com/datastrato/gravitino")
+            url.set("https://github.com/apache/gravitino")
             connection.set("scm:git:git://github.com/datastrato/gravitino.git")
```
Comment (yuqi1129) [inline build.gradle.kts:392]: do we need change here `git://github.com/datastrato/gravitino.git`?

File: dev/docker/ranger/ranger-dependency.sh (hunk 1 above)
Comment (justinmclean) [reply dev/docker/ranger/ranger-dependency.sh:27]: Reverting back for now until we can find a better way of doing this

File: build.gradle.kts (hunk 2 above)
Comment (justinmclean) [reply build.gradle.kts:392]: Yes we do

File: clients/client-python/setup.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
     version="0.6.0.dev1",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/datastrato/gravitino",
+    url="https://github.com/apache/gravitino",
     author="datastrato",
     author_email="support@datastrato.com",
```
Comment (jerryshao) [inline clients/client-python/setup.py:38]: @justinmclean can we update these two lines?

File: dev/docker/ranger/ranger-dependency.sh [hunk 4]
```diff
@@ -25,7 +25,6 @@ ranger_dir="$(cd "${ranger_dir}">/dev/null; pwd)"
 RANGER_VERSION=2.4.0
 RANGER_PACKAGE_NAME="ranger-${RANGER_VERSION}-admin.tar.gz" # Must export this variable for Dockerfile
 RANGER_DOWNLOAD_URL=https://github.com/datastrato/apache-ranger/releases/download/release-ranger-${RANGER_VERSION}/ranger-${RANGER_VERSION}-admin.tar.gz
-
```
Comment (jerryshao) [inline dev/docker/ranger/ranger-dependency.sh:28]: The change here is unnecessary, can you plewase revert it?

File: clients/client-python/setup.py (hunk 3 above)
Comment (justinmclean) [reply clients/client-python/setup.py:38]: There's another issue to do that

Comment (jerryshao) [reply clients/client-python/setup.py:38]: OK.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**